test/parallel/pytorch_parallel_test_flow_lightning.py

```diff
@@ -0,0 +1,60 @@
+from metaflow import FlowSpec, step, batch, current, pytorch_parallel, Parameter
+import pytorch_simple_reduce
+from metaflow.plugins.frameworks.pytorch import PyTorchHelper
+
+
+class PytorchLightningParallelTest(FlowSpec):
+    """
+    Test flow to test @pytorch_parallel with pytorch lightning.
+    """
+
+    num_nodes = Parameter(
+        "num_nodes",
+        help="Number of nodes in cluster",
+        default=3,
+    )
+    num_local_processes = Parameter(
+        "num_local_processes", help="Number of local processes per nodde", default=2
+    )
+
+    @step
+    def start(self):
+        self.next(self.parallel_step, num_parallel=self.num_nodes)
+
+    @pytorch_parallel
+    @step
+    def parallel_step(self):
+        """
+        Run a simple torch parallel program which learns to sum a vector and also
+        includes a simple parallel reduction to validate the parallel environment setup.
+        """
+
+        self.node_index = current.parallel.node_index
```

Comment:
Why is this line necessary?  Is this some kind of boilerplate you need to make this work?

Reply:
Good catch, this is stray code, needs to be removed.